explainer.md

```diff
@@ -0,0 +1,386 @@
+authentication problem, as:
+
+1. It requires the Account Provider to be present in the transaction (e.g. via
+   an embedded iframe), which increases user friction and lowers conversion
+   rates.
+1. The generated assertion contains no payments-related information, so it
+   cannot be used as-is to fulfill regulatory requirements to provide evidence
+   of user content (e.g. [Dynamic Linking] requirements).
+1. WebAuthn does not allow credential creation in a cross-origin iframe, thus
+   excluding a useful onboarding flow - enrolling a Customer after they have
+   completed a traditional authentication flow (e.g. via SMS OTP), without
+   the friction of redirecting them to the bank's site or app.
+
+This proposal attempts to provide an authentication solution for online
+payments that is both as strong and low-friction as WebAuthn, whilst also
+solving these three issues.
+
+### Goals
+
+Find a solution that (in no particular order):
+
+* Has as-strong or stronger authentication than current challenge flows.
+* Is less friction for the Customer than current challenge flows.
+* Is more protective of user-privacy than current frictionless flows.
+* Improves the ability for online payments to meet regulations such as [Dynamic
+  Linking].
+
+> **NOTE**: It is out-of-scope for the specification to **prove** adherence
+> to e.g. Dynamic Linking, but we aim to produce a solution that could be
+> vetted as such.
+
+* Allows for in-flow enrollment of Customers during a traditional challenge
+  flow, as well as outside of a transaction.
+
+### Non-Goals
+
+* Selection of a Payment Instrument by the Customer; it is presumed that the
+  Customer has already done so (e.g. by typing in their credit card number).
+* ID & V to establish real world identity during enrollment; it is up to the
+  Account Provider to determine the Customer's identity to their satisfaction.
+* Providing authentication for peer-to-peer or business-to-business
+  transactions.
+    * We expect that Secure Payment Confirmation may be useful in these
+      cases, but are concentrating on consumer-to-business cases for now.
+
+## Proposed Solution: Secure Payment Confirmation
+
+Secure Payment Confirmation (SPC) builds on top of [WebAuthn] to add
+payment-specific datastructures to the signed assertion, and to relax
+assumptions to allow the API to be called in payment contexts.
+
+The intention for Secure Payment Confirmation is that a Customer would enroll
+once on a given device for a given account with an Account Provider, either on
+the Account Provider's website or during a traditionally-authenticated online
+payment (e.g. after completing a challenge in a Account Provider iframe). Then,
+in subsequent transactions on **any Merchant** that wishes to use Secure
+Payment Confirmation:
+
+1. The Customer identifies themselves to the Merchant as per normal (e.g.
+   by selecting a payment instrument such as a credit card or bank account.)
```

Comment:
It is not clear to me that the goal here is for the user to identify themself to the Merchant/PSP. I think it suffices (and may avoid discussion of topics like PCI compliance) to say: 

1. The Customer selects a payment instrument (e.g., credit card or bank account).

Reply:
Done, but note: I worry folks will misconstrue 'selects' to imply that instrument selection is involved in SPC in some way.